### ISSUE TITLE
Issue transferring items in the button row 

### ISSUE THREAD

**Issue by Dantkall** [dasilva333/TowerGhostForDestiny#12]
Whenever you tap an item in the button row, the transfer bar doesn't show and usually transfers a random item in the row above it to another character. I know it was mentioned on Reddit but haven't seen it posted here.

**Comment by Dantkall** [comment]
Can't find a way to edit this, should be "bottom" row. (Auto correct via Swype)  And also I tried it again last night with the newest update and it didn't randomly move the row above it, it moved the sparrow I trapped in the bottom row to the character nearest where that button would be. So it seems like the box doesn't show up but it registers the item tap as transfer in one motion. 

I also wanted to mention because you have this on so many platforms now, I'm on Android. Can't speak to the platforms.
